### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Added German Translation
- Fixed Syntax mistakes in previous version

### CODE REVIEW

File: resources/lang/de/admin/nests.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (DaneEveritt) [inline resources/lang/de/admin/nests.php]: Its not documented, but all of the license headers can be removed from these files. We're moving towards that on all files, I just haven't done a gloabl search and replace yet.

So lines 2 - 9 can be removed.

File: resources/lang/de/admin/nests.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/**
```
Comment (DaneEveritt) [inline resources/lang/de/admin/nests.php]: Its not documented, but all of the license headers can be removed from these files. We're moving towards that on all files, I just haven't done a gloabl search and replace yet.

So lines 2 - 9 can be removed.

Comment (DRSchlaubi) [reply resources/lang/de/admin/nests.php]: I'll remove that

**Comment by DaneEveritt** [comment]
The license header should be removed from all of the files. Also any language files you have where you have no changes (e.g. the validation messages file) can be left out, it'll just default to English.

**Comment by DRSchlaubi** [comment]
I have changed that

**Comment by schrej** [comment]
The next time you merge German translation let me have a look first :sweat_smile:

**Pull request merged**